### ISSUE TITLE
Check page and locator actions for `missing-playwright-await` rule

### ISSUE THREAD

**Issue by benatshippabo** [mskelton/eslint-plugin-playwright#159]
## Suggestion

We should add additional functions to check if they are being awaited on. Currently we only check expect and test matchers, but none of the [page actions](https://playwright.dev/docs/api/class-page) or [locator actions](https://playwright.dev/docs/api/class-locator):

https://github.com/playwright-community/eslint-plugin-playwright/blob/f3cfc99f4eaaf73b9936a80f17be95668ac2efc1/src/rules/missing-playwright-await.ts#L11-L58

**Comment by revelt** [comment]
just to add 2p, for example, currently, this missing `page` `await` is not flagged up:

```js
test('has title', async ({ page }) => {
  page.goto('https://playwright.dev/'); // await is missing!
// ^^^

  // Expect a title "to contain" a substring.
  await expect(page).toHaveTitle(/Playwright/);
});
```

**Comment by mskelton** [comment]
This, the longest standing issue for eslint-plugin-playwright has finally been implemented behind a `includePageLocatorMethods` option! I'll probably make this flag true by default in the future.
